PendingReleaseNotes

```diff
@@ -319,6 +319,10 @@ CephFS: Disallow delegating preallocated inode ranges to clients. Config
   which specifies the path within the CephFS to mount this export on. If this and the other
   `EXPORT { FSAL {} }` options are the same between multiple exports, those exports will share a single CephFS client. If not specified, the default is `/`.
 
+* MGR: MGR's always-on modulues/plugins can now be force-disabled. This can be
```

Comment:
Hmm, it's under the `>=19.0.0` section. I'm not sure but my guess it's should be out in (new) `>= 19.2.2`. Looping @lflores in.

From the code perspective – LGTM.

Reply:
Yes, if it's going to land in the next point release, which is v19.2.2, let's create a new section.